app/views/admin/creator_settings/new.html.erb

```diff
@@ -5,6 +5,16 @@
     display: flex;
     flex-direction: column;
   }
+
+  .admin-help-button {
+    position: fixed;
+    bottom: var(--su-7);
+    margin-bottom: env(safe-area-inset-bottom);
+    right: var(--su-7);
+    z-index: var(--z-elevate);
+    cursor: pointer;
+    border-radius: 50px;
+  }
```

Comment:
I've reused these styles from the `admin.scss` file, as they didn't seem to be accessible within this view without adding them to the file itself. Should I approach this differently? Maybe by moving this class elsewhere?